docs/book/factories.md

```diff
@@ -126,6 +126,11 @@ support them. Create `create<type>(array $metadata)` methods for each
 type you wish to support, where `<type>` is your custom class name, minus
 the namespace.
 
+**Please note**: There is a [known issue](https://github.com/zendframework/zend-expressive-hal/issues/5) for at least Zend Router (probably
```

Comment:
The format should be changed:

```
> ### Limitation
>
> There is a [known issue](https://github.com/zendframework/zend-expressive-hal/issues/5) for at least Zend Router (probably
> others as well) for using routes with optional parameters. If in doubt, always…
```

The greater-than sign creates a blockquote and the hashes a real headline. (see this [example](https://docs.zendframework.com/zend-expressive-hal/links-and-resources/#note-on-naming))
I would also change "known issue" to "known limitation".